imap_processing/cli.py

```diff
@@ -374,7 +380,19 @@ class Codice(ProcessInstrument):
     """Process CoDICE."""
 
     def do_processing(self, dependencies):
-        """Perform CoDICE specific processing."""
+        """
+        Perform CoDICE specific processing.
+
+        Parameters
+        ----------
+        dependencies : array_like
+            List of dependencies to process.
+
+        Returns
+        -------
+        str TODO double check
```

Comment:
I think this is a list of paths it is returning.
```suggestion
        list[pathlib.Path]
```